packages/xdl/src/Webpack/WebpackInstance.ts

```diff
@@ -0,0 +1,101 @@
+import { getNameFromConfig, readConfigJsonAsync } from '@expo/config';
+import internalIp from 'internal-ip';
+import openBrowser from 'react-dev-utils/openBrowser';
+import { Urls } from 'react-dev-utils/WebpackDevServerUtils';
+
+import { printInstructions } from './createWebpackCompiler';
+import { DevServer, Protocol, WebpackSettings } from './Webpack.types';
+
+function getIP(): string {
+  return internalIp.v4.sync() || '127.0.0.1';
```

Comment:
should we use the `WEB_HOST` first if it's provided? https://github.com/expo/expo-cli/pull/1345/files#diff-85a83054c319410f82d717da5681df9fR3